VocalTractLabApi.cpp

```diff
@@ -343,18 +343,23 @@ int vtlGetConstants(int *audioSamplingRate, int *numTubeSections,
 
 
 // ****************************************************************************
-// Returns for each vocal tract parameter the minimum value, the maximum value,
-// and the neutral value. Each vector passed to this function must have at 
-// least as many elements as the number of vocal tract model parameters.
-// The "names" string receives the abbreviated names of the parameters separated
-// by spaces. This string should have at least 10*numParams elements.
+// Returns for each supra glottal parameter the minimum value, the maximum value,
+// and the standard (default) value. Each vector passed to this function must have at 
```

Comment:
"vector" is a misnomer. The type of the parameter is "array".

Reply:
done